fossunited/api/emailing.py

```diff
@@ -110,7 +133,8 @@ def create_newsletter_campaign(data: dict, event: str = None, chapter: str = Non
     newsletter_doc = frappe.get_doc(
         {
             "doctype": "Newsletter",
-            "event": event,
+            "document_type": document_type,
+            "reference_document": reference_document,
             "chapter": chapter,
```

Comment:
!!! `chapter` is passed as an argument to the function but `_chapter` is inferred within the function in case `chapter is None` so shouldn't we be using `_chapter` here?

Reply:
this was done